osd/InvalidCIDR.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Installation blocked, action required",
+    "description": "Your cluster's installation is blocked on invalid CIDR configurations. Please review the Node/Service/Pod CIDRs configuration specified during cluster provisioning and try again. For more information please refer to: https://docs.openshift.com/dedicated/4/getting_started/accessing-your-services.html#dedicated-creating-your-cluster_access",
```

Comment:
> ...and try again

Does that mean the customer needs to delete this cluster and create a new one? Or will installation re-try when values are updated? I think it's the former so we should spell that out clearly for the customer.

Reply:
Its the former.